databases/relational-databases.md

```diff
@@ -39,69 +77,132 @@ To understand, query, and insert information into a relational database, a techn
-  - examples key-value (redis), graph (Neo4j), streaming (cassandra), document (mongo)
 
-### Materials
+- [Adavntages of Relational Database](https://docs.bmc.com/docs/.../advantages-of-a-relational-database-828125922.html):
+  - Relational databases are critical to moving, sharing and maintaining the integrity of data in the modern workplace where users working with PCs, workstations, and mainframes all need access to the same data. Using an RDBMS offers the following advantages:
+     - Speed. True relational databases support client server architecture. Populate times for large ODBC databases will not grow as quickly as populate times for Paradox databases.
+     - Structured Query Language (SQL), which is the basis for relational database management systems. SQL is the standard data access language of the American National Standards Institute (ANSI) and the International Standards Organization (ISO). If you are familiar with SQL, you can access any RDBMS.
+     - Incremental data storage, which gives you a historical perspective of the data.
+     - Client/Server configuration, which lets you run Visualizer on one machine while storing data on another machine.
+     - A single interface, which provides integrated data that can be shared across platforms, networks, and devices.
+- [Describing an ORM](https://www.techopedia.com/definition/24200/object-relational-mapping--orm):
+  - Object-relational mapping (ORM) is a programming technique in which a metadata descriptor is used to connect object code to a relational database. Object code is written in object-oriented programming (OOP) languages such as Java or C#. ORM converts data between type systems that are unable to coexist within relational databases and OOP languages.
+
+- Bonus topic for the advanced: [foreign key constraints](https://en.wikipedia.org/wiki/Foreign_key)
+  - A FOREIGN KEY is a key used to link two tables together.
+  - A FOREIGN KEY is a field (or collection of fields) in one table that refers to the PRIMARY KEY in another table.
+  - Below is an example of its usage:
+    ```sql
+      CREATE TABLE Orders (
+      OrderID int NOT NULL,
+      OrderNumber int NOT NULL,
+      PersonID int,
+      PRIMARY KEY (OrderID),
+      FOREIGN KEY (PersonID) REFERENCES Persons(PersonID);
+    ```
+The table containing the foreign key is called the child table, and the table containing the candidate key is called the referenced or parent table.
+- Other types of non-relational, non-sql databases that will be encountered
+  - key-value (redis)
+    - Redis is a key-value database (also known as a key-value store) that uses a simple key/value method to store data.
+       Strings are the simplest data type in Redis and are simple key/value entries.
+  - graph (Neo4j)
+    - Neo4j is a graph database management system developed by Neo4j, Inc. Described by its developers as an ACID-compliant transactional database with native graph storage and processing. 
```

Comment:
https://github.com/JNOSQL/artemis-demo/tree/master/artemis-demo-java-se/graph-neo4j
☹️

Reply:
> I'm going to give you one more chance to double-check any files you have worked on under GSSoC and cite them.

I am really sorry for such a mistake. While checking the previous commits, I just slipped off from my mind to check the present commit.